src/services/mondomarket/index.jsx

```
@@ -0,0 +1,473 @@
+import { useEffect, useRef, useState } from "react";
+import AppInput from "../../components/form/AppInput";
+import Button from "../../components/form/Button";
+import { Spinner } from "../../components/theme/Loader";
+import Form from "../../components/form/Form";
+import { FaUpload } from "react-icons/fa";
+import ErrorMessage from "../../components/form/ErrorMessage";
+import AppSelect from "../../components/form/AppSelect";
+import AppTextArea from "../../components/form/AppTextArea";
+import { uploadImage } from "../../utils/common/cloudinary";
+
+export const AddMondoItem = ({ setAddModal, dispatch, setReload, regions }) => {
```

ESLint

[error] 'setAddModal' is missing in props validation

(react/prop-types)